### PR TITLE
Sewio updates

### PR DESCRIPTION
**Pull request by taylorcenters**
Changes from PR 148 for python3. Includes changes for sewio and apimote jamming. Does not include Dockerfile/support. Will visit this topic in a separate PR,

### CODE REVIEW

File: killerbee/__init__.py [hunk 1]
```diff
@@ -76,42 +76,39 @@ def __init__(self, device: Optional[str]=None, hardware: Optional[str]=None, dat
             else: del isSewio
 
 
-        if device is not None:
-            self.dev = device
```
Comment (rmspeers) [inline killerbee/__init__.py:80]: Where does self.dev get set now?

File: killerbee/dev_apimote.py [hunk 2]
```diff
@@ -259,18 +259,22 @@ def ping(self, da: Any, panid: Any, sa: Any, channel: Optional[int]=None, page:
...
         @type page: Integer
         @param page: Sets the subghz page, not supported on this device
         @rtype: None
         '''
+
         if self.handle is None:
             raise Exception("Handle does not exist")
 
+        if method is None or method not in ['reflexive', 'constant']:
+            raise Exception("Device does not support the provided jamming method")
```
Comment (rmspeers) [inline killerbee/dev_apimote.py]: Probably should be ValueError, or leverage the `self.capabilities.require(KBCapabilities.PHYJAM_REFLEX)` type checks.

File: killerbee/dev_sewio.py [hunk 3]
```diff
@@ -300,10 +326,37 @@ def set_channel(self, channel, page=0):
...
+        @param count: Transmits a specified number of frames, def=1
+        @type delay: Float
+        @param delay: Delay between each frame, def=0
+        @rtype: None
         '''
+
         self.capabilities.require(KBCapabilities.INJECT)
 
+        if len(packet) < 1:
+            raise Exception('Empty packet')
```
Comment (rmspeers) [inline killerbee/dev_sewio.py]: `ValueError`s or similar?

File: killerbee/dev_zigduino.py [hunk 4]
```diff
@@ -21,104 +21,103 @@
...
-        @rtype: None
-        @return: None
-        '''
-        self.capabilities.setcapab(KBCapabilities.FREQ_2400, True)
-        self.capabilities.setcapab(KBCapabilities.SNIFF, True)
-        self.capabilities.setcapab(KBCapabilities.SETCHAN, True)
-        self.capabilities.setcapab(KBCapabilities.INJECT, True)
-        #self.capabilities.setcapab(KBCapabilities.PHYJAM_REFLEX, True)
-        self.capabilities.setcapab(KBCapabilities.SET_SYNC, True)
+          '''
```
Comment (rmspeers) [inline killerbee/dev_zigduino.py]: Check indents on this? `          ` shows 10 spaces, not a 4 multiple as would expect?

File: killerbee/__init__.py (hunk 1 above)
Comment (taylorcenters) [reply killerbee/__init__.py:80]: still happens below

File: killerbee/dev_apimote.py (hunk 2 above)
Comment (taylorcenters) [reply killerbee/dev_apimote.py]: I just copied from your PR feel free to make any changes you want

File: killerbee/dev_sewio.py (hunk 3 above)
Comment (taylorcenters) [reply killerbee/dev_sewio.py]: I just copied from your PR feel free to make any changes you want

**Review by rmspeers — APPROVED** [review]

**Pull request merged**